src/index.js

```diff
@@ -2,6 +2,7 @@
 
 const bs58 = require('bs58')
 const multihashing = require('multihashing-async')
+const keyComposer = require('crypto-key-composer')
```

Comment:
Perhaps it would be better to do `const { composePrivateKey, decomposePrivateKey }`. The benefit is that it will be easier to perform tree-shaking by bundlers, as we are importing only what we need at the require level.

Reply:
Sure! I'll do that when fixing the linting errors which I'm not sure what the maintainers prefer